From 8e8c0e3f49a6684e640a40ce2db79dbbe3e92286 Mon Sep 17 00:00:00 2001
From: "algea.cao" <algea.cao@rock-chips.com>
Date: Thu, 13 Jul 2017 10:28:32 +0800
Subject: [PATCH] Revert "ARM64: dts: rk3328-evb: enable tve"

This reverts commit c69d61610a887799edcc2da4e31fbd8d699193e0.
There is conflict between cvbs and hdmi.Hdmi is used more than cvbs,
so disable cvbs by default.

Change-Id: I1c0966e0166715da0d4c11426686b28386d16a0c
Signed-off-by: algea.cao <algea.cao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328-evb.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-evb.dts b/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
index 02a974623409..f834fff885e3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-evb.dts
@@ -495,7 +495,3 @@
 &vpu_service {
 	status = "okay";
 };
-
-&tve {
-	status = "okay";
-};
-- 
2.35.3

